windows/CommonSettings.props

```diff
@@ -0,0 +1,94 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is true only for traditional make build on Linux.
For cmake doesn't enforce copy-example model (you can find guidance on link you shared bit more below).
I don't see the value in forcing people to make a copy. 
It makes CI automation a bit more complicated, and Caffe doesn't seem to be consistent across cmake/make build.